docs/content/preview/admin/yb-admin.md

```diff
@@ -1793,6 +1793,47 @@ yb-admin \
 * *replication_name*: The name of the replication to be enabled or disabled.
 * `0` | `1`: Disabled (`0`) or enabled (`1`). Default is `1`.
 
+#### get_xcluster_safe_time
+
+Reports the skew in xCluster Safe time between the laggiest tablet and the most caught up tablet. 
+
+**Syntax**
+
+```sh
+yb-admin \
+    -master_addresses <target_master_addresses> \
+    get_xcluster_safe_time \
+    [include_lag_and_skew] 
+```
+
+* *target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is `localhost:7100`.
+* *include_lag_and_skew*: Set `include_lag_and_skew` option to show `safe_time_lag_sec` and `safe_time_skew_sec`, otherwise these are hidden by default.
```

Comment:
Let's also describe what safe_time_lag_sec and safe_time_skew_sec are:

- safe_time_lag_sec - this is calculated as current time - namespace safe time.
- safe_time_skew_sec - this is calculated as the safe time of the most caught up tablet - the safe time of the least caught up tablet (ie the namespace safe time). This value can be used to determine if there are lagging tablets.

Reply:
It's already there https://github.com/yugabyte/yugabyte-db/pull/17327/files#diff-fa034eeed9f6760804ec31f0ced1dff7c33d4e54cf38155433597f9f417de9d6R1834-R1835

Reply:
Correct link: https://github.com/yugabyte/yugabyte-db/pull/17327/files#diff-fa034eeed9f6760804ec31f0ced1dff7c33d4e54cf38155433597f9f417de9d6R1862-R1863